open_academy/views/open_academy_views.xml

```diff
@@ -1,4 +1,30 @@
+<?xml version='1.0' encoding='utf-8'?>
 <odoo>
+    <record id="open_academy_dashboard_view" model="ir.ui.view">
+        <field name="name">dashboard.name</field>
```

Comment:
by `dashboard.name` I meant that you have to give it a name, not the actual text